lib/main.js

```diff
@@ -394,7 +394,7 @@ function ConfigureMathJax() {
               jax.CHTML.ex = ex = (data||defaults).ex;
               jax.CHTML.em = jax.CHTML.outerEm = em = ex / CHTML.TEX.x_height; // scale ex to x_height
               jax.CHTML.cwidth = width / em;
-              jax.CHTML.lineWidth = (linebreak ? width / em : CHTML.BIGDIMEN);
+              jax.CHTML.lineWidth = (linebreak ? width / em : 1000000);
```

Comment:
@zorkow, in the CommonHTML file, this value *is* in a variable, but it is local to the file, and can't be accessed from outside, so we can't use it here.  The `CHTML.BIGDIMEN` value doesn't exist (and never did), and that is the source of the problem.  Since it is only used once in this location, I didn't see the need to name it in `main.js` when I suggested the change to Peter earlier.